app/bundles/LeadBundle/Segment/ContactSegmentFilterCrate.php

```diff
@@ -194,7 +194,7 @@ private function setOperator(array $filter)
     {
         $operator = isset($filter['operator']) ? $filter['operator'] : null;
 
-        if ('multiselect' === $this->getType()) {
+        if ('multiselect' === $this->getType() && in_array($operator, ['in', '!in'])) {
```

Comment:
Hi, I am not sure here - `in` and `!in` have different meanings. Have you tested both scenarios?

Reply:
It's only to check if operator is equal to `in` or `!in`.
If this condition is not, all operator for multiselect is transformed to `multiselect` or `!multiselect`